nav2_sms_recovery/include/nav2_sms_recovery/sms_recovery.hpp

```diff
@@ -1,8 +1,19 @@
 // Copyright (c) 2020 Samsung Research America
-// This code is licensed under MIT license (see LICENSE.txt for details)
-
-#ifndef NAV2_SMS_RECOVEY__SMS_RECOVERY_HPP_
-#define NAV2_SMS_RECOVEY__SMS_RECOVERY_HPP_
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
You may not relicense someone elses' work.

Reply:
yes but I did not touch the copyright holder, just added the rest of standard license